assets/prometheus-k8s/rules.yaml

```diff
@@ -362,6 +362,14 @@ spec:
     rules:
     - expr: sum(openshift_build_total{job="kubernetes-apiservers",phase="Error"})/(sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Failed|Complete|Error"}))
       record: build_error_rate
+  - name: openshift-sre.rules
+    rules:
+    - expr: sum(apiserver_request_count{job="apiserver"}) by (code)
+      record: code:apiserver_request_count:sum
+    - expr: kube_pod_status_ready{condition="true",namespace="kube-system",pod=~"etcd.*"}
```

Comment:
Could we instead of reporting the raw granularity report the aggregates of each state? As in

```
sum(kube_pod_status_ready{condition="true",namespace="kube-system",pod=~"etcd.*"}) by(condition)
```

That way the cardinality of this is constantly 3 (as [condition is `true|false|unknown`](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md#pod-metrics)).

Reply:
we decided we can grab current metrics to represent this data in telemeter so this will be removed